benchmarks/pypsa/pypsa-eur-elec-20-lv1-3h-op-ucconv.yaml

```
@@ -1,1205 +1,306 @@
-    CHP: '#8a5751'
-    urban central gas CHP: '#8d5e56'
-    CHP CC: '#634643'
-    urban central gas CHP CC: '#6e4e4c'
-    CHP heat: '#8a5751'
-    CHP electric: '#8a5751'
-    district heating: '#e8beac'
-    resistive heater: '#d8f9b8'
-    residential rural resistive heater: '#bef5b5'
-    residential urban decentral resistive heater: '#b2f1a9'
-    services rural resistive heater: '#a5ed9d'
-    services urban decentral resistive heater: '#98e991'
-    urban central resistive heater: '#8cdf85'
-    retrofitting: '#8487e8'
-    building retrofitting: '#8487e8'
-    # hydrogen
-    H2 for industry: "#f073da"
-    H2 for shipping: "#ebaee0"
-    H2: '#bf13a0'
-    hydrogen: '#bf13a0'
-    retrofitted H2 boiler: '#e5a0d9'
-    SMR: '#870c71'
-    SMR CC: '#4f1745'
-    H2 liquefaction: '#d647bd'
-    hydrogen storage: '#bf13a0'
-    H2 Store: '#bf13a0'
-    H2 storage: '#bf13a0'
-    land transport fuel cell: '#6b3161'
-    H2 pipeline: '#f081dc'
-    H2 pipeline retrofitted: '#ba99b5'
-    H2 Fuel Cell: '#c251ae'
-    H2 fuel cell: '#c251ae'
-    H2 turbine: '#991f83'
-    H2 Electrolysis: '#ff29d9'
-    H2 electrolysis: '#ff29d9'
-    # ammonia
-    NH3: '#46caf0'
-    ammonia: '#46caf0'
-    ammonia store: '#00ace0'
-    ammonia cracker: '#87d0e6'
-    Haber-Bosch: '#076987'
-    # syngas
-    Sabatier: '#9850ad'
-    methanation: '#c44ce6'
-    methane: '#c44ce6'
-    # synfuels
-    Fischer-Tropsch: '#25c49a'
-    liquid: '#25c49a'
-    kerosene for aviation: '#a1ffe6'
-    naphtha for industry: '#57ebc4'
-    methanolisation: '#83d6d5'
-    methanol: '#468c8b'
-    shipping methanol: '#468c8b'
-    industry methanol: '#468c8b'
-    # co2
-    CC: '#f29dae'
-    CCS: '#f29dae'
-    CO2 sequestration: '#f29dae'
-    DAC: '#ff5270'
-    co2 stored: '#f2385a'
-    co2 sequestered: '#f2682f'
-    co2: '#f29dae'
-    co2 vent: '#ffd4dc'
-    CO2 pipeline: '#f5627f'
-    # emissions
-    process emissions CC: '#000000'
-    process emissions: '#222222'
-    process emissions to stored: '#444444'
-    process emissions to atmosphere: '#888888'
-    oil emissions: '#aaaaaa'
-    shipping oil emissions: "#555555"
-    shipping methanol emissions: '#666666'
-    land transport oil emissions: '#777777'
-    agriculture machinery oil emissions: '#333333'
-    # other
-    shipping: '#03a2ff'
-    power-to-heat: '#2fb537'
-    power-to-gas: '#c44ce6'
-    power-to-H2: '#ff29d9'
-    power-to-liquid: '#25c49a'
-    gas-to-power/heat: '#ee8340'
-    waste: '#e3d37d'
-    other: '#000000'
-    geothermal: '#ba91b1'
-    geothermal heat: '#ba91b1'
-    geothermal district heat: '#d19D00'
-    geothermal organic rankine cycle: '#ffbf00'
-    AC: "#70af1d"
-    AC-AC: "#70af1d"
-    AC line: "#70af1d"
-    links: "#8a1caf"
-    HVDC links: "#8a1caf"
-    DC: "#8a1caf"
-    DC-DC: "#8a1caf"
-    DC link: "#8a1caf"
-    load: "#dd2e23"
-    waste CHP: '#e3d37d'
-    waste CHP CC: '#e3d3ff'
-    HVC to air: 'k'
+      linearized_unit_commitment: false
```

Shouldn't this line also be under the `options:` key? E.g.:
```suggestion
  options:
      linearized_unit_commitment: false
```

---

And similarly in the other files. Is there a way we can generate new LP files given these configs, and check that it has MILP features, if you haven't already done this?

---

Thanks Sid, you're right! That issue should be present only in this file anyway, in the others we don't have unit commitment. I already generated the LP file and checked it for the presence of MILP variables, and I also sent it to Julian, you can find it at https://drive.google.com/file/d/1Y5QTw1wnjaRLDG4x2PZ2MQzamMHcOP_w/view?usp=drive_link